Automattic/wp-calypso
# Push Notifications: Add Tracks events

Adds Tracks events for:
* Notifications enabled
* Notifications disabled
* Permissions blocked
* Notification clicked
* Notification opened (requires WPCOM patch)

To test: 
* Enable debug for notifications and analytics. From your browser console: `localStorage.setItem( 'debug', 'calypso:push-notifications,calypso:analytics' );`
* Enable notifications, and make sure the event is properly recorded
* Disable notifications, and make sure the event is properly recorded
* Reset your permissions, and reload the page -- the block notifications, and make sure the event is properly recorded
* Send yourself a notification (with WPCOM patch), make sure that when the notification is received, the event is tracked (won't show up in debug... as it just loads the pixel)
* Click the notification, make sure the event is recorded

Test live: https://calypso.live/?branch=add/push-notifications/tracks
